substrate/frame/assets/src/migration/v1.rs

```diff
@@ -0,0 +1,144 @@
+// This file is part of Substrate.
+
+// Copyright (C) Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use crate::{
+	migration::v2::old::{Asset, AssetDetails},
```

Comment:
This is a bit confusing for migrations to depend on each other.

Reply:
Might be controversial but I think it's better to just duplicate it.. 
- Only a few lines of code
- No confusing import
- Can easily see the format of the storage without switching to a different file
- Not like they'll ever change, so we're not creating extra refactor work in the future
- Each migration remains standalone